backend/handlers/handlers.go

```diff
@@ -1100,7 +1100,7 @@ func (handlers *Handlers) getBitsuranceURL(r *http.Request) interface{} {
 		lang = utilConfig.MainLocaleFromNative(handlers.backend.Environment().NativeLocale())
 	}
 
-	return bitsurance.BitsuranceURL(lang, r.URL.Query().Get("code"))
+	return bitsurance.BitsuranceURL(handlers.backend.DevServers(), lang)
```

Comment:
makes me think of testnet - i you run 'BitBox -testnet`, what is the expected behavior for bitsurance? 

in `BitsuranceURL()` TBTC seems enabled, but that does not make much sense for users right?

Reply:
Good point, I removed the check for testnet accounts as they won't be supported. In testnet mode the insurance page will be available, but there will be no insurable accounts on the account selection page. I updated the last commit message to specify that testnet is not supported.